### PR TITLE
feat(config): add parameters for new Zooz ZEN72/74/77 firmwares, fix ZEN05 param 1

### LINKED ISSUE #4786: Update Zooz device files
To account for new devices, per Zooz:

Below are the new settings which have been added to the following dimmer versions so far:

ZEN72 10.10 (ver. 1.0 hardware) and 2.0 (ver. 2.0 hardware)
ZEN74 10.10 (ver. 1.0 hardware, no 2.0 hardware available at the moment)
ZEN77 10.20 (ver. 1.0 hardware), 2.10 (ver. 2.0 hardware), and 3.0 (ver. 3.0 hardware)

Please let me know if you have any questions here!

Remote Z-Wave Dimming Duration (Multilevel Commands For Group 3 And 4)

Parameter 30: Set the time it takes to get from 0% to 100% brightness on dimmers and smart bulbs directly associated with your dimmer in Groups 3 and 4 when pressing and holding the paddle (physical dimming) on your dimmer. The number entered as value corresponds to the number of seconds.
Values: 1 – 99 (seconds). Default: 5.
Size: 1 byte dec

Ramp Rate Control

Parameter 9: Adjust the ramp rate ON for your dimmer when the top paddle is pressed (physical ramp rate on). Values correspond to the number of seconds it take for the dimmer to reach full brightness when operated manually.
Values: 1 – 99 (seconds). 0 – instant on. Default: 0.
Size: 1 byte dec
 NOTE:  On all other previous firmware versions including 1.0 and 10.0, this setting was used for all ramp rate control (to adjust behavior for on/off physical and Z-Wave ramp rate). The default value for this parameter used to be 1.
 
Parameter 27: Adjust the ramp rate OFF for your dimmer when the bottom paddle is pressed (physical ramp rate off). Values correspond to the number of seconds it take for the dimmer to turn off when operated manually.
Values: 1 – 99 (seconds). 0 – instant off. Default: 2.
Size: 1 byte dec
 
Parameter 28: Adjust the ramp rate ON for your dimmer when a Z-Wave command is sent to turn the dimmer on (Z-Wave ramp rate on). Values correspond to the number of seconds it take for the dimmer to reach full brightness when operated via Z-Wave.
Values: 1 – 99 (seconds). 0 – instant on. 255 – match physical. Default: 255.
Size: 1 byte dec
 
Parameter 29: Adjust the ramp rate OFF for your dimmer when a Z-Wave command is sent to turn the dimmer off (Z-Wave ramp rate off). Values correspond to the number of seconds it take for the dimmer to turn off when operated via Z-Wave.
Values: 1 – 99 (seconds). 0 – instant off. 255 – match physical. Default: 255.
Size: 1 byte dec

### LINKED ISSUE #5248: Correct Zooz ZEN005 Parameter 1
From Zooz:

The Parameter should be as below:

Parameter 1: Decide how the LED indicator behaves depending on the on/off status of the plug. The LED indicator will flash twice to confirm a setting has been changed.

Values:

0 – LED indicator is on when the plug is on, LED indicator is off when the plug is off (default)

1 – LED indicator is on when the plug is off, LED indicator is off when the plug is on

2 – LED indicator is always off

The customer is using the recommended JS Z-Wave integration for Home Assistant and option 3 from the options listed here.

Parameter 1 is displaying with value 0 as "Always On," as shown in the screenshot below from the customer:

image.png

### PR DESCRIPTION
**Pull request by alvinchen1**
Revised https://github.com/zwave-js/node-zwave-js/pull/5156.

~~Partially addresses issue https://github.com/zwave-js/node-zwave-js/issues/4786~~
Added parameter 27, 28, 29 to zen77.
Added parameter 30 to zen72, zen74 and zen77 hardware v1.
Fix zooz zen05 param 1 https://github.com/zwave-js/node-zwave-js/issues/5248

Still no resolution for hardware versions where firmware drops below version 10.x (i.e. version 2.0, 3.0) differentiation see https://github.com/zwave-js/node-zwave-js/issues/5248 and https://github.com/zwave-js/node-zwave-js/pull/5244

But still provides updated configurations for zooz devices where firmware versions don't decrease vs. increase.

fixes: #4786 
fixes: #5248

### CODE REVIEW

File: packages/config/config/devices/0x027a/zen72.json [hunk 1]
```diff
@@ -129,6 +129,11 @@
 			"#": "29",
 			"$if": "firmwareVersion >= 10.10",
 			"$import": "templates/zooz_template.json#zwave_off_dimmer_ramp_rate"
+		},
+		{
+			"#": "30",
+			"$if": "firmwareVersion >= 10.10",
```
Comment (TaiSHiNet) [inline packages/config/config/devices/0x027a/zen72.json]: These if can be replaced with
```
"$if": "(firmwareVersion >= 10.10 && firmwareVersion < 2.0) || (firmwareVersion >= 2.10 && firmwareVersion < 3.0) || firmwareVersion >= 3.10"
```

Comment (TaiSHiNet) [reply packages/config/config/devices/0x027a/zen72.json]: This should address the issue where the firmware version "decreases"

**Review by AlCalzone — DISMISSED** [review]
Thanks for starting this, I took the liberty to solve the firmware version mess

**Comment by AlCalzone** [comment]
@zwave-js-bot fix lint

**Review by AlCalzone — APPROVED** [review]

**Comment by TaiSHiNet** [comment]
yay! Thansk @alvinchen1 and @AlCalzone !

**Pull request merged**